BlueCombing/Views/MyActivityView/BadgeGradeView.swift

```diff
@@ -12,106 +12,93 @@ struct BadgeGradeView: View {
     var user: User?
     @State var lockBadges = 8
     @State var myBadges : [String] = []
-    var body: some View {
-
-        VStack {
-            BadgeGradeViewNavbar()
-            ScrollView {
-                if let user = user {
-                    ForEach (0..<myBadges.count, id:\.self) { i in
-                        if myBadges[i] == user.representBadge {
-                            VStack() {
-                                BadgeInfoView(tempBadge: myBadges[i])
-                                    .frame(width: deviceWidth)
-                            }.frame(width: deviceWidth,height: 120)
-                                .background(Color.combingBlue2)
-                                
-                        }
-                        else {
-                            VStack() {
-                                BadgeInfoView(tempBadge: myBadges[i])
-                                    .frame(width: deviceWidth)
-                            }.frame(width: deviceWidth,height: 120)
-                                
-                        }
-                        
+    
+    var badgeGradeViewNavbar: some View {
+        HStack{
+            Image(systemName: "chevron.backward")
+                .font(.Body2)
+                .onTapGesture {
+                    dismiss()
+                }
+            Spacer()
+            Text("비치코밍 등급")
+                .font(.Body3)
+            Spacer()
+            Spacer().frame(width: 20)
```

Comment:
```suggestion
            Text("비치코밍 등급")
                .font(.Body3)
            Spacer()
            Spacer()
                .frame(width: 20)
```
어떤 것은 바로 . 뒤에 함수를 어떤 것을 줄을 바꿔서 호출을 하고 있습니다. 통일되게 부탁드립니다.